manage.sh

```diff
@@ -17,6 +17,7 @@ if [ "$command" == "install" ]; then
     source .venv/bin/activate
     pip install --upgrade pip
     pip install -r requirements.txt
+    sudo apt install graphviz
```

Comment:
this will throw an error on dists with pacman...
but I think something like `sudo apt install graphviz || pacman -S graphviz` could work
I'll test this